scripts/standard_configure_mac.sh

```diff
@@ -49,5 +49,5 @@ rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"
  -DUSE_EIGEN_MKL=OFF                                                                                        \
  -DKRATOS_GENERATE_PYTHON_STUBS=ON
 
-# Buid
+# Build
```

Comment:
Line 10 can be changed to following:
You can find a list `with` all the `compilation` options in INSTALL.md or here: